fla/ops/common/chunk_o.py

```
@@ -347,12 +381,15 @@ def chunk_bwd_kernel_dv(
         b_g_last = tl.load(g + (min(i_t * BT + BT, T) - 1) * H)
-
-        b_dv *= tl.where(m_t, exp(-b_g + b_g_last), 0)[:, None]
+    if USE_G_GAMMA:
+        b_gamma = tl.load(g_gamma + i_h)
+        b_g = b_gamma * (tl.arange(0, BT) + 1)
+        b_g_last = b_gamma * (T % BT)
 
     m_A = (o_t[:, None] <= o_t[None, :]) & (m_t[:, None] & m_t)
-    if USE_G:
+    if USE_G or USE_G_GAMMA:
         b_A = tl.where(m_A, b_A * exp(b_g[None, :] - b_g[:, None]) * scale, 0).to(do.dtype.element_ty)
```

Pylint (3.3.7)

[error] Possibly using variable 'b_g' before assignment

(E0606)